sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Program.cs

```diff
@@ -69,7 +70,7 @@ private static DigitalTwinsClient GetDigitalTwinsClient(string tenantId, string
 
             // DefaultAzureCredential supports different authentication mechanisms and determines the appropriate credential type based of the environment it is executing in.
             // It attempts to use multiple credential types in an order until it finds a working credential.
-            var tokenCredential = new DefaultAzureCredential();
+            TokenCredential tokenCredential = new DefaultAzureCredential();
```

Comment:
From TSI SDK, someone (Binal?) suggested we downcast this to the base type to make it more explicit that it could be any impl of it.